hive-hooks/src/main/java/com/expediagroup/apiary/extensions/hooks/pathconversion/converters/PathConverter.java

```diff
@@ -0,0 +1,110 @@
+/**
+ * Copyright (C) 2018-2021 Expedia, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.expediagroup.apiary.extensions.hooks.pathconversion.converters;
+
+import java.util.regex.Matcher;
+
+import org.apache.hadoop.hive.metastore.api.Partition;
+import org.apache.hadoop.hive.metastore.api.StorageDescriptor;
+import org.apache.hadoop.hive.metastore.api.Table;
+
+import lombok.extern.slf4j.Slf4j;
+
+import com.expediagroup.apiary.extensions.hooks.config.Configuration;
+import com.expediagroup.apiary.extensions.hooks.converters.GenericConverter;
+import com.expediagroup.apiary.extensions.hooks.pathconversion.config.PathConversionConfiguration;
+import com.expediagroup.apiary.extensions.hooks.pathconversion.models.PathConversion;
+
+@Slf4j
+public class PathConverter extends GenericConverter {
+
+  public PathConverter(Configuration configuration) {
+    super(configuration);
+  }
+
+  @Override
+  public PathConversionConfiguration getConfiguration() {
+    return (PathConversionConfiguration) super.getConfiguration();
+  }
+
+  /**
+   * Converts a path for the given Table.
+   *
+   * @param table Table location to potentially alter.
+   * @return true if table location is altered, false otherwise.
+   */
+  @Override
+  public boolean convertTable(Table table) {
+    if (!getConfiguration().isPathConversionEnabled()) {
+      log.trace("[{}-Filter] pathConversion is disabled. Skipping path conversion for table.",
```

Comment:
```suggestion
      log.trace("pathConversion is disabled. Skipping path conversion for table.",
```
You can add the class name as an element to be output via the configuration of whatever logging framework you are using. Is there a reason to do that again here?

Reply:
Not in the current state, this looks like an artifact of one of the previous iterations where this made sense to have. Removed